docs/victoriametrics-cloud/quickstart.md

```diff
@@ -193,19 +194,26 @@ When you already have at least one deployment you can create a new one by clicki
 
 On the opened screen, choose parameters of your new deployment:
 
-* `Deployment type` 
+* `Deployment name` – a unique name for your deployment;
+* `Global settings` – choose between:
   * Single - for affordable, performant single-node deployments;
   * Cluster - for highly available and multi-tenant deployments;
-* `Region` – AWS region where deployment will run;
-* Desired `storage capacity` for storing metrics (you always can expand disk size later);
-* `Retention` period for stored metrics.
-* `Size` of your deployment [based on your needs](https://docs.victoriametrics.com/guides/understand-your-setup-size.html)
+  * Cloud provider - for deployments on AWS, GCP, Azure, etc.
+  * Region - cloud provider region where deployment will run;
+* `Tier and storage` – choose between:
```

Comment:
we should copy hints from (i) icons to docs

Reply:
added information about recommended capcity